cypress/integration/course-page.test.ts

```diff
@@ -0,0 +1,56 @@
+import { HttpMethod } from "@lib/http-method";
+import {
+  TEST_BOOK,
+  TEST_COURSE,
+  TEST_DEPARTMENT,
+} from "cypress/support/constants";
+import { MAX_NUM_POSTS } from "@lib/helpers/constants";
+
+
+describe("Course page", () => {
+  const COURSE_CODE = TEST_DEPARTMENT.abbreviation + "-" + TEST_COURSE.code;
+
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
+
+  it("should allow a user to navigate to a book page", () => {
+    cy.visit(
+      "/course/" + TEST_DEPARTMENT.abbreviation + "-" + TEST_COURSE.code
+    );
+    cy.get('div[href*="book/"]').first().click()
```

Comment:
Consider calling `cy.get('[href*="book/"]')` instead to make the test more resilient to change